src/app/session-lobby/session-lobby.component.ts

```diff
@@ -9,13 +11,22 @@ import { TeacherSessionService } from '../teacher-session.service';
 export class SessionLobbyComponent implements OnInit {
 
   sessionID: string;
+  roundData: FirebaseRound;
 
-  constructor(public teacherSessionService: TeacherSessionService) {
+  constructor(public teacherSessionService: TeacherSessionService, public teacherRoundService: TeacherRoundService) {
     this.sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
+    this.roundData = {flowerSpeciesIds: ['asclepias_syriaca', 'coreopsis_palmata'],
+                          status: 'fine',
+                          running: false,
+                          currentTime: 0, };
   }
```

Comment:
```suggestion
  constructor(public teacherSessionService: TeacherSessionService, public teacherRoundService: TeacherRoundService) {  }
```

You can probably simplify this a bit by assigning these two fields where they're declared.